build.gradle

```diff
@@ -16,4 +20,82 @@ dependencies {
     compile 'org.apache.httpcomponents:httpclient:4.5.3'
     testCompile 'junit:junit:4.12'
     testCompile 'org.json:json:20180813'
+}
+
+bintray {
+    user = System.getenv('BINTRAY_USER')
+    key = System.getenv('BINTRAY_API_KEY')
+
+    pkg {
+        repo = 'maven'
+        name = archivesBaseName
+        licenses = ['Apache-2.0']
+        vcsUrl = 'https://github.com/snowplow-referer-parser/java-referer-parser'
+    }
+    publications = ['mavenJava']
+    version {
+        name = version
+        gpg {
+            sign = true
```

Comment:
I don't think we have a key so this will most likely fail

Reply:
It's obligatory to have signing enabled. 

https://github.com/bintray/gradle-bintray-plugin#Maven_Central_Sync
> (...) also files must be signed to be sent to Maven Central, so GPG file signing should be enabled (see above) if Maven Central sync is enabled

Here is an article how to create a signature:
https://medium.com/@nmauti/publishing-a-project-on-maven-central-8106393db2c3